### PR TITLE
Updates from package template

### PR DESCRIPTION
**Pull request by Cadair**
This PR has been generated by a script, it should update the repo with the latest changes from the package template.

### CODE REVIEW

File: .cruft.json [hunk 1]
```diff
@@ -10,6 +10,12 @@
       "author_name": "The SunPy Community",
       "author_email": "sunpy@googlegroups.com",
       "project_url": "https://sunpy.org",
+      "github_repo": "sunpy/drms",
+      "sourcecode_url": "https://github.com/sunpy/drms",
+      "download_url": "https://pypi.org/project/sunpy",
```
Comment (Cadair) [inline .cruft.json:15]:
```suggestion
      "download_url": "https://pypi.org/project/sunraster",
```

File: pyproject.toml [hunk 2]
```diff
@@ -113,7 +120,72 @@ write_to = "drms/_version.py"
       name = "Documentation Fixes"
       showcontent = true
```
Comment (CyclingNinja) [inline pyproject.toml:122]: merge artifacts

**Comment by nabobalis** [comment]
pre-commit.ci autofix

**Comment by nabobalis** [comment]
docs plus online will fail due to JSOC.

**Review by nabobalis — APPROVED** [review]

**Pull request merged**